### PR TITLE
fix address type function

### PR DESCRIPTION
**Pull request by yushih**

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscript] ⚠️ `yushi/fix-address-type-func` doesn't full our branch naming convention.
Should follow `<tag>/<jira-ticket-id>/<short-name>`
<details>
<summary>Details</summary>

- `<tag>`: `fix`, `feat`, `chore`, `refactor`, `test`, ...
- `<jira-ticket-id>`: The issue ID of JIRA all capital letters
- `<short-name>`: A short and descriptive name for the branch
</details>

[](__JIRA_COMMENT_FLAG)

**Comment by neuodev** [comment]
@yushih  it is already fixed #3124 here. Feel free to close the PR

File: packages/yoroi-extension/app/api/ada/lib/storage/bridge/utils.js [hunk 1]
```diff
@@ -331,14 +331,11 @@ export function isJormungandrAddress(
...
-
-  return kind in types;
+  if (kind === CoreAddressTypes.JORMUNGANDR_SINGLE) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_GROUP) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_ACCOUNT) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_MULTISIG) return true;
+  return false;
 }
 
 export function isErgoAddress(
```
Comment (neuodev) [inline packages/yoroi-extension/app/api/ada/lib/storage/bridge/utils.js:341]: I think this solution is more cleaner 
```suggestion
const types = [
    CoreAddressTypes.JORMUNGANDR_SINGLE,
    CoreAddressTypes.JORMUNGANDR_GROUP,
    CoreAddressTypes.JORMUNGANDR_ACCOUNT,
    CoreAddressTypes.JORMUNGANDR_MULTISIG,
  ];
  return  types.some(addrType => addrType === kind);
}
```

Comment (yushih) [reply packages/yoroi-extension/app/api/ada/lib/storage/bridge/utils.js:341]: @AhmedIbrahim336 or you can use `types.includes(kinds)`

**Pull request closed without merging**